styles/AdvancedEditor.css

```diff
@@ -172,12 +172,16 @@
   padding-bottom: 6px;
   text-align: left;
 }
+.ProseMirror .box-element .block-section_title h1 {
+  font-style: italic !important;
+  font-weight: 400 !important;
+  line-height: 24px; /* 114.286% */
+  letter-spacing: -0.369px;
+}
```

Comment:
we need to apply that just for the first title, so will need to use a child selector `<`.

I don't have a strong opinion  but we need to apply that style just for placeholder `h1.empty-node[data-placeholder]::before`, that is up to you

Reply:
done